generators/server/generator.mjs

```diff
@@ -222,10 +224,20 @@ export default class JHipsterServerGenerator extends BaseApplicationGenerator {
         await this.composeWithJHipster(GENERATOR_COMMON);
       },
 
-      async composeLanguages() {
+      async composing() {
+        const { buildTool, enableTranslation } = this.jhipsterConfigWithDefaults;
+        if (buildTool === GRADLE) {
```

Comment:
Do you think that we can use derived constant right after the seed phase?

Reply:
Config files are supposed to be dirty at this point.
It's possible to create a class with getters for this purpose.
Or investigate the pros and cons of moving composing priority to after preparing priority.

Eg: the server generator changes the webappSourceDir and webappTestDir (dotnet blueprint), at preparing webappAppDir can be created with `${webappSourceDir}/app`. It may be be outdated if composing is done after preparing.